packages/SwingSet/test/liveslots-helpers.js

```diff
@@ -14,13 +14,19 @@ import {
   makeBringOutYourDead,
 } from './util.js';
 
-export function buildSyscall() {
+export function buildSyscall(skipLogging) {
```

Comment:
Will take a closer look at the test on Wednesday. Just at a quick glance: why this new option?

Also why change all the `log.push` instead of just changing the thing that reads the array?

Reply:
You should make these comments on #5207, which is the PR containing the commits you're comment on here.

It's so the array can be absent when it's not being used, which could be significant for a new class of tests that aren't ever going to look at the log contents (which record _all_ syscall activity), especially for tests that are large or long running.